### DISCUSSION TITLE
您好,请问能不能实现Popover过一些时间自动隐藏?

### DISCUSSION THREAD

**Discussion by Broderick890** [tusen-ai/naive-ui#4347] [Q&A]
光标在内容范围内,过一些时间Popover自动隐藏,非常需要这个功能,**感谢大佬!**

**Comment by XieZongChen** [comment]
![image](https://user-images.githubusercontent.com/46394163/213072414-b036e7d7-2392-430e-9a5c-009909107ac6.png)
使用非受控模式，自己实现计时器，一段时间后调用这个方法

**Reply by Broderick890** [reply]
非常感谢!
